pkg/products/grafana/reconciler.go

```diff
@@ -60,26 +42,17 @@ func (r *Reconciler) GetPreflightObject(_ string) k8sclient.Object {
 }
 
 func (r *Reconciler) VerifyVersion(installation *integreatlyv1alpha1.RHMI) bool {
-	return version.VerifyProductAndOperatorVersion(
-		installation.Status.Stages[integreatlyv1alpha1.InstallStage].Products[integreatlyv1alpha1.ProductGrafana],
-		string(integreatlyv1alpha1.VersionGrafana),
-		string(integreatlyv1alpha1.OperatorVersionGrafana),
-	)
+	return true
```

Comment:
this is pretty much pointless IMO, can we not verify the grafana version by checking the VersionGrafana?

Reply:
I added comment to Verify function. Maybe will need Jira to retrieve version from running Grafana. It could be follow-on. For now seems to me it's ok.
Thanks for comment

Reply:
Cool, please create follow up jira